assets/app/scripts/app.js

```diff
@@ -160,6 +160,26 @@ angular
   })
   .constant("API_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).api))
   .constant("AUTH_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).auth))
+  .factory("ProxyPod", function(API_CFG) {
```

Comment:
ProxyPod should probably be grouped with the extension that is using it until it is either more general-use or we feel like multiple things need it